File: src/Coop.Api/Models/JsonContent.cs
using Newtonsoft.Json.Linq;
using System;

namespace Coop.Api.Models
{
    public class JsonContent
    {
        public Guid JsonContentId { get; private set; }
        public JObject Json { get; private set; }
        public JsonContent(string type, JObject json)
        {

            Json = json;
        }

        public JsonContent(JObject json)
        {
            Json = json;
        }

        protected JsonContent()
        {

        }
    }
}

File: src/Coop.App/src/app/@api/models/json-content.ts
// Copyright (c) Quinntyne Brown. All Rights Reserved.
// Licensed under the MIT License. See License.txt in the project root for license information.

import { JsonContentName } from "@api/models/json-content-name";

export type JsonContent = {
    jsonContentId?: string,
    name?: JsonContentName,
    json?: any,
    type?: any
};

